Portfolio/index.css

```diff
@@ -1 +1,64 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+      color: #515050;
+          margin: 0%;

```

Comment:
Consider using 'margin: 0;' and 'padding: 0;' instead of 'margin: 0%;' and 'padding: 0%;' for consistency with other declarations